tests/unit/test_cube.py

```diff
@@ -519,3 +519,22 @@ def but_the_fraction_is_NaN_for_unfiltered_count_zero(self):
         population_fraction = measures.population_fraction
 
         np.testing.assert_equal(population_fraction, expected_value)
+
+
+class DescribeMeanMeasure(object):
+    def it_knows_basic_measure_shape_when_it_has_no_subvars(self, _base_measure_shape):
+        expected_shape = (3, 4)
+        _base_measure_shape.return_value = expected_shape
+        measure = _MeanMeasure(None, None)
+        assert measure.shape == expected_shape
+
+    def but_it_augments_measure_shape_when_it_has_subvars(self, _base_measure_shape):
+        base_shape = (3, 4)
+        _base_measure_shape.return_value = base_shape
+        measure = _MeanMeasure(None, None, subvariables=["0001", "0002"])
+        expected_shape = base_shape + (2,)  # augmented by n_subvars
+        assert measure.shape == expected_shape
```

Comment:
Can you collapse this 2 tests onto 1? I mean using `@pytest.mark.parametrize` ... something like:

```python
    @pytest.mark.parametrize(
        "base_shape, subvariables, expected_shape",
        (((3, 4), None, (3, 4)), ((3, 4), ["0001", "0002"], (3, 4, 2))),
    )
    def it_knows_its_altered_shape(
        self, base_shape, subvariables, expected_shape, _base_measure_shape
    ):
        _base_measure_shape.return_value = base_shape
        measure = _MeanMeasure(None, subvariables=subvariables)
        assert measure.shape == expected_shape
```

Reply:
done